### PR TITLE
docs: RouterMeta typing example

### PR DESCRIPTION
**Pull request by pikax**
added to the chinese docs and also recommending a file name `typings.d.ts` to make a bit clear on where it should be placed

### CODE REVIEW

**Comment by posva** [comment]
Does it need to be in `d.ts` file? I always thought it could go along your routes declaration in the `router.ts` file

**Comment by pikax** [comment]
> Does it need to be in d.ts file? I always thought it could go along your routes declaration in the router.ts file

No, it does not need to, I personally use `.d.ts` for declarations only, should we just change it to `routes.ts` or just remove the file?

File: docs/guide/advanced/meta.md [hunk 1]
```diff
@@ -56,8 +56,8 @@ router.beforeEach((to, from) => {
 It is possible to type the meta field by extending the `RouteMeta` interface:
 
 ```ts
+// typings.d.ts
 import 'vue-router'
-
```
Comment (posva) [inline docs/guide/advanced/meta.md:60]: let's add back the empty line here and below, I find it cleaner

File: src/types/index.ts [hunk 2]
```diff
@@ -253,6 +253,8 @@ export interface _RouteRecordBase extends PathParserOptions {
  * @example
  *
  * ```ts
+ * // typings.d.ts
```
Comment (posva) [inline src/types/index.ts:256]:
```suggestion
 * // typings.d.ts or router.ts
 *
```

File: docs/zh/guide/advanced/meta.md [hunk 3]
```diff
@@ -56,6 +56,8 @@ router.beforeEach((to, from) => {
 可以通过扩展 `RouteMeta` 接口来输入 meta 字段：
 
 ```ts
+// typings.d.ts
```
Comment (posva) [inline docs/zh/guide/advanced/meta.md:59]:
```suggestion
// typings.d.ts or router.ts

```

File: docs/guide/advanced/meta.md [hunk 4]
```diff
@@ -56,8 +56,8 @@ router.beforeEach((to, from) => {
 It is possible to type the meta field by extending the `RouteMeta` interface:
 
 ```ts
+// typings.d.ts
 import 'vue-router'
```
Comment (posva) [inline docs/guide/advanced/meta.md:60]:
```suggestion
// typings.d.ts or router.ts
import 'vue-router'

```

**Review by posva — CHANGES_REQUESTED** [review]

File: docs/zh/guide/advanced/meta.md [hunk 5]
```diff
@@ -56,6 +56,9 @@ router.beforeEach((to, from) => {
 可以通过扩展 `RouteMeta` 接口来输入 meta 字段：
 
 ```ts
+// typings.d.ts or router.ts
+
+import 'vue-router'
```
Comment (posva) [inline docs/zh/guide/advanced/meta.md:61]:
```suggestion
// typings.d.ts or router.ts
import 'vue-router'

```

File: src/types/index.ts [hunk 6]
```diff
@@ -253,6 +253,9 @@ export interface _RouteRecordBase extends PathParserOptions {
  * @example
  *
  * ```ts
+ * // typings.d.ts or router.ts
+ *
+ * import 'vue-router';
```
Comment (posva) [inline src/types/index.ts:258]:
```suggestion
 * // typings.d.ts or router.ts
 * import 'vue-router';
 *
```

**Review by posva — APPROVED** [review]
Thanks! I think it's nice to tell people they can put it right next to their `createRouter()` but also move it to a `d.ts` file.

**Pull request merged**